### ISSUE TITLE
Nevada online documents have moved

### ISSUE THREAD

**Issue by smrazgs** [ngds/metadata-issues#55]
Arrrrgh!
see  http://www.nbmg.unr.edu/dox/.  
Links in metadata no longer work e.g. http://search.geothermaldata.org/dataset/map-showing-distribution-and-structure-of-magnasite-deposits-southwest-of-overton-clark-county-/resource/40fab6c4-52e7-410f-bff0-a2e96bc54df8

Need to talk to nevada hub about redirecting existing URLs

**Comment by Lbookman** [comment]
I'll try and see on couch how many have changed and if I can do a easy fix. @smrazgs

**Comment by smrazgs** [comment]
yes, if you look for urls that start with http://www.nbmg.unr.edu/dox/ I'd expect those to be the broken ones.

**Comment by Lbookman** [comment]
there are 541 records with that link. Did anyone from Nevada get back to you?  @smrazgs

**Comment by Lbookman** [comment]
Email Rachel over at UNR

**Comment by smrazgs** [comment]
hopefully there's some simple way to update the metadata; did we harvest it from them? they'll need to update their metadata if so.  A redirect on their server would be the best thing for them to do, so if other's have linked to the papers, those links work
